packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -47,6 +47,8 @@ const Trade = ({
     const [try_synthetic_indices, setTrySyntheticIndices] = React.useState(false);
     const [is_digits_widget_active, setIsDigitsWidgetActive] = React.useState(false);
 
+    const extra_small_device = React.useMemo(() => window.innerHeight < 480, [window]);
```

Comment:
should we memoize this? 
If the orientation is changed to `landscape` it will still consider the device as extra small